bin/cigar

```diff
@@ -18,7 +18,7 @@ use Brunty\Cigar\Outputter;
 use Brunty\Cigar\Parser;
 use Brunty\Cigar\Result;
 
-$options = getopt('c:ia:u:jh:', ['version', 'help', 'quiet', 'config:', 'insecure', 'auth:', 'url:', 'json', 'header:']);
+$options = getopt('c:ia:u:jh:t:ct:', ['version', 'help', 'quiet', 'config:', 'insecure', 'auth:', 'url:', 'json', 'header:', 'timeout:', 'connect-timeout:']);
```

Comment:
Shortopts are for single-character options only, so `ct` won't work (we also already have a `c` and a `t` option, so it'll produce results like this:

```
> php opts.php -c 'config_file.json' -t 'timeout' -ct '1'
array(2) {
  ["c"]=>
  array(2) {
    [0]=>
    string(16) "config_file.json"
    [1]=>
    string(1) "t"
  }
  ["t"]=>
  string(7) "timeout"
}
```

Reply:
Doh my bad, changed it :+1: